linera-views/src/views/key_value_store_view.rs

```diff
@@ -40,6 +40,106 @@ static KEY_VALUE_STORE_VIEW_HASH_RUNTIME: LazyLock<HistogramVec> = LazyLock::new
     .expect("Histogram can be created")
 });
 
+#[cfg(with_metrics)]
+/// The runtime of get
+static KEY_VALUE_STORE_VIEW_GET_RUNTIME: LazyLock<HistogramVec> = LazyLock::new(|| {
+    prometheus_util::register_histogram_vec(
+        "key_value_store_view_get_runtime",
+        "KeyValueStoreView get runtime",
+        &[],
+        Some(vec![
+            0.001, 0.003, 0.01, 0.03, 0.1, 0.2, 0.3, 0.4, 0.5, 0.75, 1.0, 2.0, 5.0,
```

Comment:
@afck @andresilva91 Should we have a function in `prometheus_util` to generate these vectors?

Reply:
Yes, I agree! That would also make the buckets more consistent.